pkg/claims/output.go

```diff
@@ -0,0 +1,97 @@
+package claims
+
+import (
+	"sort"
+
+	"get.porter.sh/porter/pkg/cnab"
+	"get.porter.sh/porter/pkg/storage"
+	"github.com/cnabio/cnab-go/bundle"
+	"github.com/cnabio/cnab-go/bundle/definition"
+)
+
+var _ storage.Document = Output{}
+
+type Output struct {
+	Name         string `json:"name"`
+	Namespace    string `json:"namespace"`
+	Installation string `json:"installation"`
+	RunID        string `json:"runId"`
+	ResultID     string `json:"resultId"`
+	Value        []byte `json:"value"`
+}
+
+func (o Output) DefaultDocumentFilter() interface{} {
+	return map[string]interface{}{"resultId": o.ResultID, "name": o.Name}
+}
+
+func (o Output) ToCNAB() cnab.Output {
+	return cnab.Output{
```

Comment:
Oops, it's because that function isn't needed. I'll remove it and the others. Right now the only thing that needs it is Run -> Claim, and perhaps later Installation as well.